### PR TITLE
testing/razercfg: new aport

### PR DESCRIPTION
**Pull request by tanertas**
https://bues.ch/cms/hacking/razercfg.html
Razer mouse configuration tool.

### CODE REVIEW

**Comment by tanertas** [comment]
@andypost can you take a look at this?

File: testing/razercfg/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+source="
+	https://github.com/mbuesch/razer/archive/$pkgname-$pkgver.tar.gz
+	razerd.initd
+	"
+options="!check" # No test suite
+builddir="$srcdir/razer-$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	cmake . -DCMAKE_INSTALL_PREFIX='/usr'
```
Comment (clandmeter) [inline testing/razercfg/APKBUILD]: You are not running make here?
I think make install is not triggering it which is not the correct location.

**Comment by clandmeter** [comment]
Please also rebase against master so CI picks it up.

File: testing/razercfg/APKBUILD (hunk 1 above)
Comment (tanertas) [reply testing/razercfg/APKBUILD]: I'm not sure what is your confusion bu I have to say that his is the cmake equivalent of --prefix=/usr.

**Comment by tanertas** [comment]
> Please also rebase against master so CI picks it up.

Why rebase? There's no conflict with master anyway since this is a new aport PR. CI already triggered when I update PR anytime.
Btw, I updated initscript.

**Comment by tanertas** [comment]
> > Please also rebase against master so CI picks it up.
> 
> Why rebase? There's no conflict with master anyway since this is a new aport PR. CI already triggered when I update PR anytime.
> Btw, I updated initscript.

@clandmeter I realized that you were referring to the "drone" CI specifically. Sorry for misunderstood. I rebased to master.

File: testing/razercfg/APKBUILD (hunk 1 above)
Comment (maxwell-k) [reply testing/razercfg/APKBUILD]: Maybe I can explain a little more:

The README says that both cmake and make are used for the build step. cmake creates the Makefile. make uses the Makefile to build the binaries. https://github.com/mbuesch/razer#building

make install is correctly called in the package step. If the binary files are not available, so if make has not been called in the build step, make will notice and compile them. Compilation should not happen in the package step, so we're suggesting you add a make call to build().

**Comment by Ikke** [comment]
Is this significant?

> WARNING: ldconfig failed: 1
Your system will probably be unable to locate librazer.so library

**Pull request closed without merging**